generators/generator-base.js

```diff
@@ -230,7 +230,7 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
     getPrettierExtensions() {
         let prettierExtensions = 'md,json,yml';
         if (!this.skipClient && !this.jhipsterConfig.skipClient) {
-            prettierExtensions = `${prettierExtensions},js,ts,tsx,css,scss`;
+            prettierExtensions = `${prettierExtensions},js,ts,tsx,vue,css,scss`;
```

Comment:
mhhh let me think. With this, I suppose we'll have `vue` in Angular or React project. Can we only include it for Vue application ? Similar to the following codes with Java ?

Reply:
We could also include `tsx` file extension only for React application

Reply:
@murdos : mhhh in this PR or, better to do a new one?

Reply:
@murdos Had some thought in mind, currently its all in a single command.

Reply:
Fixed for vue clients in this PR

Reply:
@pascalgrimaud Fixed for VUE in this PR